src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -412,6 +412,9 @@
         <summary>Gets the optional "readonly" keyword.</summary>
       </PropertyComment>
     </Field>
+    <Field Name="ScopedKeyword" Type="SyntaxToken" Optional="true">
+      <Kind Name="ScopedKeyword"/>
+    </Field>
```

Comment:
part of the type syntax, and not part of the parameter syntax?

Reply:
The feature supports *both* `scoped ref Span<int> param` and `ref scoped Span<int> param` and they mean different things. Haven't looked closely at this PR yet but it doesn't surprise me that Scoped is part of the ParameterSyntax directly.